src/test/test.cpp

```diff
@@ -675,7 +685,13 @@ saveTestTxMeta(stdfs::path const& dir)
             Json::Value& hashes = fileRoot[testCasePair.first];
             for (auto const& h : testCasePair.second)
             {
-                hashes.append(binToHex(h));
+                uint64_t tmp = h;
+                std::vector<uint8_t> buf;
+                for (size_t i = sizeof(uint64_t); i > 0; --i)
+                {
+                    buf.emplace_back(uint8_t(tmp >> (8 * (i - 1))));
```

Comment:
pretty sure we need to & 0xFF here to avoid tripping verifiers (I know Visual C++ catches this by default, I imagine valgrind might as well)

Reply:
Ok, fixed.